Tag 1.0-stable

alaxalves opened the pull request:
## Mudanças Propostas

Gerando versão estável do ambiente de Produção.

drianne committed:
Adding delete project acceptance test

Signed-off-by: MatheusRich <matheusrichardt@gmail.com>

alaxalves committed:
Merging pull request #218 from fga-gpp-mds/ts08_testes_de_aceitação

danieloda committed:
Merge branch 'devel' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into us23_planejar_sprint

MatheusRich committed:
Merging pull request #214 from fga-gpp-mds/US19_visualizar_velocity

drianne committed:
Removing unnecessary comment

danieloda committed:
Fixing html

cantuariavc committed:
Adding date validations to Releases and Sprints

Signed-off-by: MatheusRich <matheusrichardt@gmail.com>

MatheusRich committed:
Adding date validations to edit Sprint

Signed-off-by: Vinícius Cantuária <cantuariavc@gmail.com>

cantuariavc committed:
Merging branch 'devel' into ts_06_refatorar_componentes

pedrokelvin committed:
Changing Burndown graph to ChartJs

Signed-off-by: MateusO97 <mat97_@hotmail.com>
Signed-off-by: danieloda <danieloda35@gmail.com>

MateusO97 committed:
Adding ideal line to burndown

Signed-off-by: pedrokelvin <pedrokelvin2006@gmail.com>
Signed-off-by: danieloda <danieloda35@gmail.com>

MatheusRich committed:
Fixing edit Release bug

Signed-off-by: cantuariavc <cantuariavc@gmail.com>

ThalissonMelo committed:
Merge remote-tracking branch 'origin/us23_planejar_sprint' into us23_planejar_sprint

cantuariavc committed:
Fixing edit Release bug

Signed-off-by: MatheusRich <matheusrichardt@gmail.com>

MatheusRich committed:
Fixing delete Release bug

Signed-off-by: Vinícius Cantuária <cantuariavc@gmail.com>

cantuariavc committed:
Fixing Release Card number

Signed-off-by: MatheusRich <matheusrichardt@gmail.com>

cantuariavc committed:
Adding Realises Cards scrolling in Releases

Signed-off-by: MatheusRich <matheusrichardt@gmail.com>

cantuariavc committed:
Adding no content Sprints in Releases

Signed-off-by: MatheusRich <matheusrichardt@gmail.com>

MatheusRich committed:
Refactoring Releases component

MatheusRich committed:
Refactoring SprintCard component

Matheus456 committed:
Adding validation to Add and Edit Project

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

Matheus456 committed:
Adding validation to Add and Edit Sprint

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

alaxalves committed:
Merging pull request #207 from fga-gpp-mds/us23_planejar_sprint

drianne committed:
Adding issues graphic component

Signed-off-by:  matheus456<matheus456456@gmail.com>

Matheus456 committed:
Adding issues graphic component to project

Matheus456 committed:
Adding issues graphic route

cantuariavc committed:
Fixing logging

Signed-off-by: MatheusRich <matheusrichardt@gmail.com>

cantuariavc committed:
Fixing login

Signed-off-by: MatheusRich <matheusrichardt@gmail.com>

MateusO97 committed:
Adding validations to Add and Edit Issue

MateusO97 committed:
Adding validations to Add and Edit Releases

MateusO97 committed:
Fixing validations in Sprint component

MateusO97 committed:
Fixing variable name

Signed-off-by: pedrokelvin <pedrokelvin2006@gmail.com>
Signed-off-by: danieloda <danieloda35@gmail.com>

drianne committed:
Adding condition if date interval is not setted

MatheusRich committed:
Changing deprecated localStorage calls to vuex

MatheusRich committed:
Changing deprecated localStorage calls to vuex

matheussbernardo committed:
Adding unit tests

ThalissonMelo committed:
Fixing request adress in homolog

alaxalves committed:
Merging pull request #228 from fga-gpp-mds/Fixing_Request_Adress

cantuariavc committed:
Merging branch 'devel' into ts_06_refatorar_componentes

cantuariavc committed:
Merging branch 'devel' into issue_183_validacao_periodos

alaxalves committed:
Merging pull request #223 from fga-gpp-mds/issue_183_validacao_periodos

danieloda committed:
Merge branch 'devel' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into issue_199

Matheus456 committed:
Adding arrow images to IssuesGraphic

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

Matheus456 committed:
Adding Dashboard of issues to Project page

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

Matheus456 committed:
Fixing decimal datas to integer

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

Matheus456 committed:
Increasing the size of the names in the interaction and placing the y-axis to start at zero

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

MatheusRich committed:
Adding Sprint unit tests

cantuariavc committed:
Fixing indentation and style sheet

matheussbernardo committed:
Merge branch 'devel' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into issue_front_unit_tests

cantuariavc committed:
Creating add Issue test

cantuariavc committed:
Replacing local storages to vuex

cantuariavc committed:
Replacing locais storages to vuex

matheussbernardo committed:
Fixing test when merged with devel

ThalissonMelo committed:
Merge pull request

Passando gráfico burndown para a biblioteca ChartJS

Matheus456 committed:
Adding new arrows images to IssuesGraphic

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

cantuariavc committed:
Merging branch 'issue_front_unit_tests' into issue_231_testes_70%_cobertura

Matheus456 committed:
Changing visualizate of comparation of issues opened and issues closed on diferent intervals

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

danieloda committed:
Improving project page design

Signed-off-by: Daniel Oda <danieloda35@gmail.com>

danieloda committed:
Merge branch 'devel' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into newProjectPage

matheussbernardo committed:
Adding unit test for navbar component

matheussbernardo committed:
Merge branch 'issue_231_testes_70%_cobertura' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into issue_231_testes_70%_cobertura

MatheusRich committed:
Adding Sprint Review unit tests

danieloda committed:
Merge branch 'newProjectPage' into US24_visualizar_grafico_de_issues

ThalissonMelo committed:
Merge pull request

Adicionando testes de unidade

MatheusRich committed:
Removing GPA and SearchBard components

MatheusRich committed:
Fixing acceptance tests

matheussbernardo committed:
Improving List Sprint Review tests

danieloda committed:
Adding title to backlog screen

MatheusRich committed:
Merging pull request #213 from fga-gpp-mds/issue_199

ThalissonMelo committed:
Fixing backlog list issues

ThalissonMelo committed:
Fixing kanban to alocate issues to sprints correctly

Matheus456 committed:
Fixing percentage calculation

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

Matheus456 committed:
Merging branch 'US24_visualizar_grafico_de_issues' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into US24_visualizar_grafico_de_issues

cantuariavc committed:
Refactoring User Profile

Matheus456 committed:
Fixing Codeclimate issue

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

alaxalves committed:
Merging pull request #242 from fga-gpp-mds/issue_241_remover_componentes_inúteis

alaxalves committed:
Fixing typo in .travis.yml

alaxalves committed:
Merging branch 'devel' into issue_246_travis_typo

alaxalves committed:
Changing homolog env to a non-fucked up one

ThalissonMelo committed:
Making modifications to refresh issue list on kanban

ThalissonMelo committed:
Making modifications to refresh issue list on kanban

MatheusRich committed:
Merge remote-tracking branch 'origin/ts_06_refatorar_componentes' into ts_06_refatorar_componentes

Matheus456 committed:
Merging branch 'devel' into US24_visualizar_grafico_de_issues

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

Matheus456 committed:
Remove unusable botton and deacreasing siz of closed issues doughnut

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

MateusO97 committed:
Merge branch 'devel' into ts09_validacao_frontend

MateusO97 committed:
Removing login test since validations changes

MateusO97 committed:
Fixing Date validations in Releases

MateusO97 committed:
Improving date validations in Sprints components

cantuariavc committed:
Adding User Profile unit tests

MatheusRich committed:
Merge branch 'devel' into ts_06_refatorar_componentes

MatheusRich committed:
Merging pull request #247 from fga-gpp-mds/issue_246_travis_typo

alaxalves committed:
Merging branch 'devel' into issue_238_homolog_env

MatheusRich committed:
Merging pull request #248 from fga-gpp-mds/issue_238_homolog_env

MateusO97 committed:
Fixing password confirmation in Register

MateusO97 committed:
Refactoring should not register test

MatheusRich committed:
Fixing unit tests

matheussbernardo committed:
Adding issues unit tests

matheussbernardo committed:
Merge branch 'issue_231_testes_70%_cobertura' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into issue_231_testes_70%_cobertura

matheussbernardo committed:
Improving unit test on user profile, stories, sprintreview

Matheus456 committed:
Fixing operator '==' to '==='

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

MateusO97 committed:
Fixing mouse cursor in navbar logo

cantuariavc committed:
Fixing Code Climate issues

Signed-off-by: MatheusRich <matheusrichardt@gmail.com>

ThalissonMelo committed:
Merge branch 'issue243_Fixing_kanban' of github.com:fga-gpp-mds/Falko-2017.2-FrontEnd into issue243_Fixing_kanban

alaxalves committed:
Merging pull request #245 from fga-gpp-mds/US24_visualizar_grafico_de_issues

alaxalves committed:
Merging pull request #244 from fga-gpp-mds/issue243_Fixing_kanban

alaxalves committed:
Merging pull request #253 from fga-gpp-mds/issue_250_cursor_logo

cantuariavc committed:
Adding scroll in Sprints Cards

MatheusRich committed:
Fixing unit tests

ThalissonMelo committed:
Fixing github slug of project

alaxalves committed:
Merging pull request #255 from fga-gpp-mds/issue_254_github_slug

MatheusRich committed:
Restyling scrollbar

cantuariavc committed:
Mergeing branch 'devel' into ts_06_refatorar_componentes

MatheusRich committed:
Improving layout

cantuariavc committed:
Refactoring Project component

matheussbernardo committed:
Adding spinner on import github projects component

matheussbernardo committed:
Fixing bug where forcing / route would go to homepage even if logged

matheussbernardo committed:
Fixing code climate linter problems

cantuariavc committed:
Adding Release date validation

cantuariavc committed:
Fixing Code Climate issues and Routes

MatheusRich committed:
Adding acceptance test to NotFound component

cantuariavc committed:
Removing Release and Sprints components

MatheusRich committed:
Fixing router

matheussbernardo committed:
Fixing spinner on import github projects

matheussbernardo committed:
Fixing layout of import repos modal

alaxalves committed:
Merging pull request #256 from fga-gpp-mds/issue_216_sessao_homepage

cantuariavc committed:
Removing GitHub attribute

cantuariavc committed:
Merging branch 'devel' into issue_251_remove_github_nickname

MatheusRich committed:
Fixing acceptance test

alaxalves committed:
Merging pull request #259 from fga-gpp-mds/issue_251_remove_github_nickname

MatheusRich committed:
Adding tutorial to import repository

MatheusRich committed:
Fixing import repository button

danieloda committed:
Fixing wrong title appearance

MatheusRich committed:
Restyling modals

danieloda committed:
Fixing dashboard apearance in non github projects

cantuariavc committed:
Fixing GitHub Project unit tests

cantuariavc committed:
Fixinig Add Release unit test

cantuariavc committed:
Merging branch 'devel' into ts_06_refatorar_componentes

Matheus456 committed:
Adding metrics on Sprint page

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

cantuariavc committed:
Fixing issues reported by Álax

matheussbernardo committed:
Starting points on kanban

cantuariavc committed:
Fixing GitHub linkation

Signed-off-by: MatheusRich <matheusrichardt@gmail.com>

alaxalves committed:
Merging pull request #263 from fga-gpp-mds/issue_258

alaxalves committed:
Merge branch 'devel' into ts_06_refatorar_componentes

alaxalves committed:
Merging pull request #252 from fga-gpp-mds/ts_06_refatorar_componentes

alaxalves committed:
Merging pull request #264 from fga-gpp-mds/issue_260

MatheusRich committed:
Adding link to github on issue number

MatheusRich committed:
Merge branch 'devel' into issue_261_linking_account_tutorial

MatheusRich committed:
Fixing condition to bind class on button

MatheusRich committed:
Changing modal min size

Matheus456 committed:
Modifying Issues and Projects components to not show the graph if has not a problems or related git account

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

matheussbernardo committed:
Improving kanban

Matheus456 committed:
Changing color and legend of Burndown

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

Matheus456 committed:
Fixing size of dashboard for zoom equal 100%

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

MatheusRich committed:
Merging pull request #271 from fga-gpp-mds/issue_270_arrumar_burndown

Matheus456 committed:
Stylizing component Metrics and remove method click

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

MatheusRich committed:
Merging pull request #273 from fga-gpp-mds/issue_272_arrumar_tamanho_dashboard

cantuariavc committed:
Adding scroll in GitHub Projects

MatheusRich committed:
Merging pull request #262 from fga-gpp-mds/issue_261_linking_account_tutorial

cantuariavc committed:
Adding minimum size in scrolls divs

MatheusRich committed:
Restyling add release button

MatheusRich committed:
Merging pull request #277 from fga-gpp-mds/issue_268_modal_size

MatheusRich committed:
Merging pull request #281 from fga-gpp-mds/issue_280_add_release_button

alaxalves committed:
Merging pull request #269 from fga-gpp-mds/issue_267_mensagem_grafico

matheussbernardo committed:
Improving kanban cards

MatheusRich committed:
Fixing border right issue

Matheus456 committed:
Creating component AddGrades

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

Matheus456 committed:
Adding AddGrades to Project

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

alaxalves committed:
Removing unecessary d3js dependencies

MatheusRich committed:
Merging pull request #283 from fga-gpp-mds/issue_282_border_right

MatheusRich committed:
Merge branch 'devel' into issue_237_mover_metricas

MatheusRich committed:
Merging pull request #285 from fga-gpp-mds/issue_278_d3js

alaxalves committed:
Merging pull request #266 from fga-gpp-mds/issue_237_mover_metricas

MartinsLucas committed:
Creating grade component

MartinsLucas committed:
Merge branch 'US21_manter_nota' of github.com:fga-gpp-mds/Falko-2017.2-FrontEnd into US21_manter_nota

Matheus456 committed:
Merging branch 'devel' into issue_128_tela_spinnet

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

MateusO97 committed:
Adjusting useless routes

Signed-off-by: Pedro Kelvin <pedrokelvin2006@gmail.com>

MartinsLucas committed:
Merge branch 'devel' of github.com:fga-gpp-mds/Falko-2017.2-FrontEnd into US21_manter_nota

MatheusRich committed:
Merging pull request #291 from fga-gpp-mds/issue185_rotas_inuteis

matheussbernardo committed:
Improving kanban with assignees

matheussbernardo committed:
Merge branch 'devel' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into issue_257_Pontuacion_and_assignee

Matheus456 committed:
Fixing acceptance tests

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

MartinsLucas committed:
Merge branch 'devel' of github.com:fga-gpp-mds/Falko-2017.2-FrontEnd into US21_manter_nota

matheussbernardo committed:
Improving kanban assignees

Matheus456 committed:
Fixing acceptance tests

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

Matheus456 committed:
Merging branch 'US21_manter_nota' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into US21_manter_nota

cantuariavc committed:
Fixing first Release Card remove

Matheus456 committed:
Removing prints on GitHubProjects

Signed-off-by: Matheus Roberto <matheus456456@gmail.com>

alaxalves committed:
Fixing CodeClimate issues

MatheusRich committed:
Merging pull request #293 from fga-gpp-mds/issue_284_remocao_ultima_release

alaxalves committed:
Removing unnecessary console.log()

matheussbernardo committed:
Updating charts when clicked on tab

matheussbernardo committed:
Merge branch 'issue_257_Pontuacion_and_assignee' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into issue_257_Pontuacion_and_assignee

alaxalves committed:
Merging pull request #292 from fga-gpp-mds/US21_manter_nota

matheussbernardo committed:
Clearing localstorage on logout

matheussbernardo committed:
Removing some console.log

Commit:
Merging pull request #295 from fga-gpp-mds/issue_257_Pontuacion_and_assignee

Resolving #257

ThalissonMelo committed:
Merging request #298 from fga-gpp-mds/issue_286_zerar_estado_vuex

matheussbernardo committed:
Merge branch 'devel' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into ts09_validacao_frontend

matheussbernardo committed:
Fixing tests that use veevalidate

matheussbernardo committed:
Fixing cypress tests

cantuariavc committed:
Merging branch 'devel' into issue_231_testes_70%_cobertura

cantuariavc committed:
Fixing unit testes

cantuariavc committed:
Fixing Code Climate issues

cantuariavc committed:
Fixing link verification for GitHub account

matheussbernardo committed:
Fixing vue-spinner cypress

MatheusRich committed:
Merging pull request #227 from fga-gpp-mds/ts09_validacao_frontend

matheussbernardo committed:
Fixing spinner

cantuariavc committed:
Fixing Codacy issues

alaxalves committed:
Merging pull request #290 from fga-gpp-mds/issue_128_tela_spinnet

cantuariavc committed:
Merging remote-tracking branch 'origin/devel' into issue_231_testes_70%_cobertura

cantuariavc committed:
Fixing unit tests

cantuariavc committed:
Removing Add e Edit Issues

alaxalves committed:
Merging pull request #279 from fga-gpp-mds/issue_231_testes_70%_cobertura

alaxalves commented:
@MatheusRich Tu tem como desabilitar a checagem do Codacy nesse PR, parece que o Codacy não roda na _master_

MatheusRich committed:
Translating README.md to english

alaxalves committed:
Fixing README typo

alaxalves committed:
Merging pull request #308 from falko-org/readme_translation

MatheusRich committed:
Translating Pull Request Template to english

MatheusRich committed:
Adding confirmation before logout

MatheusRich committed:
Improving Cypress logout test

Adds an assertion to the content of the window confirm. Plus, it adds and ID to the login button to make testing easier

MatheusRich committed:
Fixing logout unit test

MartinsLucas committed:
Merge pull request #315 from falko-org/logout_confirmation

Adição de confirmação antes do logout

MartinsLucas committed:
Merge pull request #310 from falko-org/translating_pull_request_template

Tradução do Template de Pull Request

MatheusRich committed:
Translating issue template to english

MatheusRich committed:
Fixing typo

MatheusRich committed:
Updating README.md

MatheusRich committed:
Fixing sprint creation and  date validation

MatheusRich committed:
Fixing Release date validation

MatheusRich committed:
Adding col class to div

MatheusRich committed:
Adding link to user profile

MatheusRich committed:
Removing useless code

alaxalves committed:
Merging pull request #326 from falko-org/issue_314

alaxalves committed:
Merging pull request #324 from falko-org/issue_316

alaxalves committed:
Merging pull request #323 from falko-org/sprint_creation

alaxalves committed:
Merging pull request #321 from falko-org/issue_template_translation

alaxalves committed:
Bumping npm packages to prevent security warns

MatheusRich committed:
Merge pull request #334 from falko-org/issue_330_broken_dependencies

Bumping npm packages

alaxalves commented:
@MatheusRich This may be accepted now, dont it?

MatheusRich commented:
@alaxalves I don't think this PR was made from the 1.0 tag. Don't you think this would be somewhat weird/inconsistent?

alaxalves committed:
Fixing warn dependencie warn

alaxalves committed:
Deleting unecessary scripts and changing scripts locations

alaxalves committed:
Removing unnecessary line in PR template

As discussed in https://github.com/falko-org/Falko-API/pull/227#discussion-diff-185677362R18

Signed-off-by: Alax Alves <alaxallves@gmail.com>

alaxalves committed:
Removing unnecessasry volume in travis' docker

alaxalves committed:
Travis now is using default docker-compose script

alaxalves committed:
Modularing docker & scripts

alaxalves committed:
Adding sample Makefile

MatheusRich committed:
Merge pull request #337 from falko-org/issue-330-broken-dependencies

Fixing warn dependencies

alaxalves committed:
Creating .github dir and organizing templates in it

MatheusRich committed:
Merge pull request #339 from falko-org/issue-331-makefile

Adding sample Makefile

alaxalves committed:
Translating templates

alaxalves committed:
Merge pull request #338 from falko-org/issue-333-rm-scripts

jrkong committed:
Fixed broken link to CONTRIBUTING.md in README.md

alaxalves committed:
Merge pull request #341 from jrkong/readme-fix

Fixed broken link to CONTRIBUTING.md in README.md
